projects/storefrontstyles/scss/components/misc/chat-messaging/_messaging.scss

```diff
@@ -0,0 +1,139 @@
+%cx-messaging {
+  .container {
+    display: flex;
+    flex-direction: row;
+    padding: 15px 0;
+    min-height: 400px;
+    padding-bottom: 2rem;
+
+    .cx-avatar-line {
+      border: 0px solid;
+      margin: 0 0 0 2.1875rem;
+      color: #d3d6db;
+      border-right-width: 1px;
+    }
+
+    .cx-messages {
+      display: flex;
+      flex-direction: column;
+      width: 100%;
+      gap: 1rem;
+      margin: 0 0 0 -2.1875rem;
+      max-height: 400px;
+      overflow-y: auto;
+
+      label {
+        margin-bottom: 0;
+        font-size: var(--cx-font-size, 0.875rem);
+      }
+
+      .cx-message-card {
+        display: flex;
+        flex-direction: row;
+        gap: 0.875rem;
+
+        div {
+          width: 100%;
+        }
+      }
+
+      .cx-message-display {
+        @include media-breakpoint-up(md) {
+          padding: 0 0 0 4.875rem;
+        }
+      }
+
+      .cx-message-left-align-text {
+        padding: 20px 28px;
+        border: 1px solid;
+        background-color: #fff1f1;
+        border-color: #d3d6db;
+        border-radius: 0px 16px 16px 16px;
+      }
+
+      .cx-message-right-align-text {
+        padding: 20px 28px;
+        border: 1px solid;
+        background-color: #f1f1f1;
+        border-color: #d3d6db;
+        border-radius: 0px 16px 16px 16px;
+      }
+
+      .cx-attachment {
+        cx-icon {
+          color: #a1aabf;
+        }
+        margin: 0;
+        color: var(--cx-color-text);
+      }
+    }
+  }
+
+  .cx-message-footer {
+    display: flex;
+    gap: 1rem;
+
+    @include media-breakpoint-up(md) {
+      padding: 0 0 0 4.875rem;
+    }
+
+    @include media-breakpoint-down(sm) {
+      flex-direction: column;
+      .cx-send {
+        align-self: flex-end;
+      }
+    }
+
+    .cx-message-box {
+      width: 100%;
+
+      cx-file-upload {
+        display: flex;
+        align-items: flex-start;
+        justify-content: left;
+        width: 100%;
+        button {
+          width: auto;
+        }
+        .cx-message-footer-text {
+          text-decoration: underline;
+          cursor: pointer;
+          padding: 0rem 0.5rem;
+          white-space: nowrap;
+        }
+        cx-icon {
+          color: #a1aabf;
+          cursor: pointer;
+          font-size: 1.25rem;
+        }
+      }
+
+      .cx-footer-label {
+        font-weight: 600;
```

Comment:
`font-weight: var(--cx-font-weight-bold);`

Reply:
adding `font-weight: var(--cx-font-weight-semi);` as it matches more with the mockup